tough/src/cache.rs

```diff
@@ -42,7 +42,7 @@ impl<'a, T: Transport> Repository<'a, T> {
                 self.cache_target(&targets_outdir, target_name.as_ref())?;
             }
         } else {
-            let targets = &self.targets.signed.targets;
+            let targets = &self.targets.signed.get_targets_map();
```

Comment:
Stylisticly in keeping with what I see in Rust land, they don't seem to have the `get_` prefix.

```suggestion
            let targets = &self.targets.signed.targets_map();
```